lxd/request/requestor.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/url"
 	"slices"
 
+	"github.com/canonical/lxd/lxd/cluster/request"
```

Comment:
This package contains only the clienttype.go file with these definitions. Can we just move them here so we only have one `request` package?

Reply:
happy with that if it doesn't enlarge the import size of lxd-agent, but its likely already importing request package anyway